schema_utils_test.py

```diff
@@ -480,6 +529,29 @@ def test_list_schema_with_len(self):
         invalid_vals = [['1', 13], {'a': 'b'}, {}, None, 123, 'abc', ['c'], []]
         self.check_normalization(schema, mappings, invalid_vals)
 
+    def test_html_schema(self):
+        """Tests for valid html schema, an html string. Note that
+        html.cleaner() is called in normalize_against_schema.
```

Comment:
I'm not sure what to do with this note. Perhaps add tests that include dangerous elements like script tags?

In general don't just look at coverage, make sure to write enough tests to convince yourself that this function would not go wrong in the future (imagine you can't see the code behind it).